{"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"hierarchy":{"paths":[["doc:\/\/SwiftPy\/documentation\/SwiftPy","doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError"]]},"kind":"article","metadata":{"modules":[{"name":"SwiftPy"}],"roleHeading":"API Collection","title":"LocalizedError Implementations","role":"collectionGroup"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftpy\/pythonerror\/localizederror-implementations"]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/LocalizedError-Implementations"},"topicSections":[{"anchor":"Instance-Properties","generated":true,"title":"Instance Properties","identifiers":["doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/errorDescription","doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/failureReason","doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/helpAnchor","doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/recoverySuggestion"]}],"references":{"doc://SwiftPy/documentation/SwiftPy/PythonError":{"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"PythonError","kind":"identifier"}],"kind":"symbol","abstract":[],"navigatorTitle":[{"text":"PythonError","kind":"identifier"}],"type":"topic","url":"\/documentation\/swiftpy\/pythonerror","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError","role":"symbol","title":"PythonError"},"doc://SwiftPy/documentation/SwiftPy/PythonError/errorDescription":{"identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/errorDescription","title":"errorDescription","kind":"symbol","role":"symbol","abstract":[],"url":"\/documentation\/swiftpy\/pythonerror\/errordescription","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"?"}],"type":"topic"},"doc://SwiftPy/documentation/SwiftPy/PythonError/failureReason":{"role":"symbol","abstract":[],"title":"failureReason","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"failureReason","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"type":"topic","kind":"symbol","url":"\/documentation\/swiftpy\/pythonerror\/failurereason","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/failureReason"},"doc://SwiftPy/documentation/SwiftPy/PythonError/recoverySuggestion":{"role":"symbol","abstract":[],"title":"recoverySuggestion","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"recoverySuggestion","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"type":"topic","kind":"symbol","url":"\/documentation\/swiftpy\/pythonerror\/recoverysuggestion","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/recoverySuggestion"},"doc://SwiftPy/documentation/SwiftPy/PythonError/helpAnchor":{"role":"symbol","url":"\/documentation\/swiftpy\/pythonerror\/helpanchor","type":"topic","kind":"symbol","title":"helpAnchor","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"helpAnchor","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/helpAnchor","abstract":[]},"doc://SwiftPy/documentation/SwiftPy":{"role":"collection","url":"\/documentation\/swiftpy","abstract":[{"text":"A fast and lightweight Python interpreter with Swift macro binding tools.","type":"text"}],"type":"topic","title":"SwiftPy","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy","kind":"symbol"}}}